plugins/gcloud/lib/samson_gcloud/image_scanner.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+module SamsonGcloud
+  class ImageScanner
+    WAITING = 0
+    SUCCESS = 1
+    FOUND = 2
+    ERROR = 3
+
+    FINISHED = [SUCCESS, FOUND].freeze
+
+    # time to wait for scan to complete since we cannot tell pending from success
+    # TODO: find a way to actually tell success from pending
+    GRACE_PERIOD = 10.minutes
+
+    class << self
+      # found by inspecting
+      # gcloud alpha container images list-tags gcr.io/$PROJECT_ID/base/alpine --show-occurrences --log-http
+      def scan(build)
```

Comment:
Shall we cache the results of this for a while, so we don't re-fetch the same information on every page load?

Reply:
the result is cached on the build in a column